index.html

```diff
@@ -29,13 +29,8 @@
                         "Please enter a name for the new document:",
                     )
                     var newDoc = {id: 42, name: name, content: ""} // FIXME
-                    setActive(newDoc.id)
                     setDocuments([...documents, newDoc])
-
-                    // Doesn't seem to work?
-                    setCurrentContent(
-                        documents.find((d) => d.id == newDoc.id).content,
-                    )
```

Comment:
If `currentContent` is derived state from `documents` and `active`, this is not necessary.

- `setDocuments` sets the content and
- `setActive` makes it current